src/components/document-id/DocumentId.vue

```diff
@@ -94,7 +95,7 @@
 import Vue from 'vue'
 import axios from 'axios'
 import { Component, Emit, Prop, Watch } from 'vue-property-decorator'
-import { SessionStorageKeys } from '@/enums/sbc-common-components-constants'
+import { SessionStorageKeys } from '@/enums'
```

Comment:
Will this still work when this shared component is built into a UI? (Will this enum be present in `@/enums` in the UI?)

Reply:
Yessir, will add the export to the enum index file.
The pattern seems to be consolidating the enums exports from the index, whether they are local/imported etc 
Seems best for reuse.

Reply:
In Storybook (ie, in the current project), `@/enums` will find the new enum locally.

In, say, Create UI, `@/enums` will point to Create UI's enums folder -- but this enum isn't in that folder.

Option 1: change this to `from '@bcrs-shared-components/enums'.
Option 2: in Create UI (or wherever you need the enum), re-export the enum in enums/index.ts.

I prefer option 1, if it works with Storybook.

Reply:
It didn't work previously but i did add it to the enums/index here as well. I can try option 1 with this change.

Reply:
I believe some other shared components import shared enums and interfaces. Hopefully it works 🤞

Reply:
Totally, it should now that sbc-constants were added to the bcrs enums/index!

Reply:
Updated